packages/grid/src/vaadin-grid.js

```diff
@@ -179,6 +179,7 @@ import { StylingMixin } from './vaadin-grid-styling-mixin.js';
  * `expanded-row`         | Expanded row
  * `selected-row`         | Selected row
  * `details-opened-row`   | Row with details open
+ * `loading-row`          | Row that is waiting for data from data provider
```

Comment:
How do you style, for example, cells of a loading row (using the native `::part()`)? I guess there's going to be part names for the cells like `loading-row-cell`? At that point, we could consider removing `loading-row` etc since the `<tr>` elements really shouldn't be styled?

Reply:
Good point, I didn't think about that. 

@rolfsmeds @jouni Do you think we should add this part, or use `loading-cell` on the cells instead?

Reply:
The same concerns all the other row states as well, like `expanded`, `selected`...

Reply:
Thanks. I will add these to the original issue and update the PR for cells accordingly (#5020).

Reply:
Seems that something like `vaadin-grid::part(last-frozen-cell)::part(body-row-cell)` wouldn't work either.

Reply:
There is support for `::part(part-name variant-name)` in some browsers at least, though I can't find any authorative information on it. There is an [issue](https://github.com/w3c/csswg-drafts/issues/3502) in the spec repo, however the [spec](https://www.w3.org/TR/css-shadow-parts-1/#part) itself only mentions a single identifier, while [MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/::part) mentions multiple ones.

Reply:
The following snippet works for me in Chrome, Firefox and Safari 15.6:

```css
vaadin-grid::part(body-cell frozen-cell) {
  background-color: #e2e2e2;
}
```